lib/ret/enums.ex

```diff
@@ -9,3 +9,4 @@ defenum(Ret.SceneListing.State, :scene_listing_state, [:active, :delisted], sche
 defenum(Ret.Avatar.State, :avatar_state, [:active, :removed], schema: "ret0")
 defenum(Ret.AvatarListing.State, :avatar_listing_state, [:active, :delisted, :removed], schema: "ret0")
 defenum(Ret.Asset.Type, :asset_type, [:image, :video, :model], schema: "ret0")
+defenum(Ret.Hub.Privacy, :hub_privacy, [:public, :private, :invite_only], schema: "ret0")
```

Comment:
hmm, this name is OK, I was thinking `access_policy` though.

Reply:
upon reading further, I forgot about `entry_mode` -- perhaps we should consolidate these states (ie, add `closed` to the new enum and update references to `entry_mode`)